src/utils.ts

```diff
@@ -6,7 +6,7 @@ import type {
 } from './JsonRpcEngine';
 
 /**
- * **ATTN:** Assumes that only one of the `result` and `error` properties is
+ * ATTN:** Assumes that only one of the `result` and `error` properties is
```

Comment:
Looks like something messed up the bolding on these comments. The `**` following `ATTN:` should presumably be removed at least?